tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/fragment/FragmentConvertor.java

```diff
@@ -47,8 +47,7 @@
    */
   private static final Class<?>[] DEFAULT_FRAGMENT_PARAMS = { ByteString.class };
 
-  public static Class<? extends Fragment> getFragmentClass(Configuration conf, String dataFormat)
-  throws IOException {
+  public static Class<? extends Fragment> getFragmentClass(Configuration conf, String dataFormat) {
```

Comment:
This function just finds and returns a proper implementation of the fragment interface. Why do we need to throw IOException?